index.html

```diff
@@ -49,18 +49,27 @@
   </head>
 
   <body>
-    <h1 id="header">Basics: Beat That! 🚀</h1>
+    <h1 id="header">Welcome to Beat That 🎲</h1>
     <div id="container">
-      <p>Input:</p>
-      <input id="input-field" />
-      <br />
-      <button id="submit-button">Submit</button>
-      <p>Output:</p>
-      <div id="output-div"></div>
+      <body>
+          Create a two-digit number by selecting the order of your dice rolls.
+          <br />
+          The player with the highest number wins!
+        </p>
+
+        👇 Click <b>submit</b> to start the game.
+        <p>Input:</p>
+        <input id="input-field" />
+        <br />
+        <button id="submit-button">Submit</button>
+        <p>Current state:</p>
+        <div id="output-div"></div>
+      </body>
```

Comment:
Generally, you shouldn't put a `<body>` tag within another `<body>` tag. a standard `<p>` tag or a `<div>` tag would suffice here.